docs/ocat_target_table.rst

```diff
@@ -0,0 +1,157 @@
+PUBLIC_AVAIL                  Date publicly available
+READOUT_DETECTOR              Detector (which HRC detector or string of actual ACIS ccds)
+DATAMODE                      Instrument data mode
+JOINT                         Joint observatories (string)
+HST                           HST time (orbits?)
+NOAO                          NOAO time
+NRAO                          NRAO time
+RXTE                          RXTE time
+SPITZER                       SPITZER time
+SUZAKU                        SUZAKU time
+XMM                           XMM time
+SWIFT                         SWIFT time
+NUSTAR                        NUSTAR time
+CATEGORY                      Science category
+SEG_MAX_NUM                   
+PROP_TITLE                    Proposal Title
+PI_NAME                       Principal investigator name
+OBSERVER                      Observer name
+APP_EXP                       Approved exposure time (ks)
+EXP_TIME                      Actual exposure time (ks)
+RA                            Right Ascension
+Dec                           Declination
+SOE_ROLL                      Roll from SOE
+TIME_CRIT                     Time critical flag
+Y_OFF                         Y offset 
+Z_OFF                         Z offset
+X_SIM                         X SIM
+Z_SIM                         Z SIM
+RASTER                        Raster flag
+OBJ_TYPE                      Object type
+OBJ                           Solar system object name
+PHOTO                         Photometry flag
+VMAG                          V Mag of object
+EST_CNT_RATE                  Estimated count rate
+Forder_CNT_RATE               First order count rage
+COUNT_RATE                    
+EVENT_COUNT
+DITHER                        Dither flag
+Y_AMP                         Dither Y amplitude if custom dither
+Y_FREQ                        Dither Y frequency if custom dither
+Y_PHASE                       Dither Y phase if custom dither
+Z_AMP                         Dither Z amplitude if custom dither
+Z_FREQ                        Dither Z frequency if custom dither
+Z_PHASE                       Dither Z phase if custom dither
+ROLL                          Roll constraint flag
+WINDOW                        Window constraint flag
+UNINT                         Uninterrupt constraint flag
+POINTING_UPDATE               Pointing update constraint flag
+MONITOR                       Monitor series flag
+PRE_ID                        Obsid of previous observation in monitor series
+MON_MIN                       Min days from pre_id for monitor observation
+MON_MAX                       Max days from pre_id for monitor observation
+GROUP_ID                      Database group id
+CONSTR
+EPOCH
+PERIOD
+PSTART
+PS_MARG
+PEND
+PE_MARG
+MULTITEL
+MULTITEL_OBS
+MULTITEL_INT
+CONSTR_RMK                    Constraint in remarks flag
+TOO_TYPE
+TOO_START
+TOO_STOP
+ALT_GROUP
+ALT_TRIG
+SIMODE                        Science Instrument (SI) mode
+HRC
+SPECT_MODE
```

Comment:
??

Reply:
Not a misspelling of ASPECT.  I assume this is some HRC thing but I'm not that HRC fluent.
```
In [10]: np.unique(dat['SPECT_MODE'])                                           
Out[10]: 
<Column name='SPECT_MODE' dtype='str12' length=4>
       (DEF)
           0
     IMAGING
NORMAL (DEF)
```